src/tests/performance/ping_pong.rs

```diff
@@ -0,0 +1,208 @@
+    // ┌─────────┬────────────┬────────────┬────────────┬────────────────┬────────────┬────────────┬────────────┬────────────┬────────────┬────────────────┬────────────┬──────────────┐
+    // │  peers  │  requests  │  min (ms)  │  max (ms)  │  std dev (ms)  │  10% (ms)  │  50% (ms)  │  75% (ms)  │  90% (ms)  │  99% (ms)  │  completion %  │  time (s)  │  requests/s  │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │       1 │        150 │          0 │         51 │              5 │          0 │          0 │          0 │          0 │         51 │         100.00 │       0.39 │       384.53 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      10 │        150 │          0 │         50 │              4 │          0 │          0 │          0 │          0 │          0 │         100.00 │       2.13 │       705.79 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      15 │        150 │          0 │         51 │              5 │          0 │          0 │          0 │          0 │         41 │          14.80 │      12.13 │        27.45 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      20 │        150 │          0 │         10 │              2 │          0 │          0 │          0 │          0 │         10 │          19.17 │      13.95 │        41.22 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      30 │        150 │          0 │         50 │              7 │          0 │          0 │          0 │          0 │         45 │          14.60 │      15.83 │        41.52 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      50 │        150 │          0 │         45 │              3 │          0 │          0 │          0 │          0 │          0 │          10.13 │      19.42 │        39.14 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      75 │        150 │          0 │         57 │              6 │          0 │          0 │          0 │          0 │         41 │           8.23 │      25.28 │        36.63 │
```

Comment:
> let synth_counts = vec![1, 10, 15, 20, 30, 50];

We removed 75 from here, so I think we should refresh the table here?

Reply:
Updated.